swagger.yaml

```diff
@@ -378,6 +392,18 @@ definitions:
               - email
           dataset_id:
             type: string
+          dimensions:
+            type: array
+            items:
+              type: object
```

Comment:
Just wondering why we are doing this as an embedded object rather than a reference (as per highlight in the next field as an example). Not sure it's a problem just questioning it.

Reply:
Good point Jess - I've updated to reflect this.